about.html

```diff
@@ -0,0 +1,158 @@
+        <li>
+          <a href="#" class="topLinks">
+            <i class="bi bi-facebook"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="topLinks">
+            <i class="bi bi-twitter"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#" class="topLinks">English</a>
+        </li>
+        <li>
+          <a href="#" class="topLinks">My Page</a>
+        </li>
+        <li>
+          <a href="#" class="topLinks">Logout</a>
+        </li>
+      </ul>
+      <nav class="nav_bar">
+        <a href="./index.html" class="mylogo"
+          ><img src="./images/logo.png" alt=""
+        /></a>
+        <ul class="navbar_container">
+          <li class="navbar_items">
+            <a href="./index.html" class="nav_links">Home</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">Progarm</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">Join</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">Sponsor</a>
+          </li>
+          <li class="navbar_items">
+            <a href="#" class="nav_links">News</a>
+          </li>
+          <li class="navbar_items">
+            <a class="nav-bttn" href="#">UNAMA Campaign</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="hamburger">
+        <span class="lines one"></span>
+        <span class="lines two"></span>
+        <span class="lines three"></span>
+      </div>
+    </header>
+    <!-- Main sectiom  -->
+    <main class="container-fluid main-about">
+      <section class="row justify-content-center">
+        <div class="col-md-11 text-md-center col-11 p-1 m-2 content-wrapper">
+          <p class="short-para text-center">"Afghanistan Development"</p>
+          <h1 class="main-heading-about text-center">
+            Afghanistan <br />
+            Conference Geneva <br />
+            2020
+          </h1>
+          <p class="info-about text-center p-md-5">
+            The 2020 Afghanistan Conference was a pledging conference, co-hosted
+            by the Governments of Afghanistan and Finland with the United
+            Nations, that took place November 2020. The 2020 Afghanistan
+            Conference was a pledging conference, co-hosted by the Governments
+            of Afghanistan and Finland with the United Nations, that took place
+            November 2020.
+          </p>
+          <p class="place-info-about text-center">
+            @ Geneva Finland, Organized by the Government of Afghanistan
+          </p>
+        </div>
+      </section>
+    </main>
+    <section class="row justify-content-center p-md-5 p-5">
+      <div class="col-md-4 col-8 text-center p-4">
+        <img class="footer-logo" src="./images/logo.png" alt="" />
+      </div>
+      <div>
+        <h2 class="text-center p-4 p-md-4 pb-md-1">
+          See The Past 2020 Afghanistan Conference
```

Comment:
# Dekstop Size: About Page
- It contains a **Middle** section with a centered column that includes a section **header, a divider, a paragraph of text, and an image or large logo.**
- It contains a **BOTTOM** section with a centered column that includes a section **header, a divider, a line of text**, and two images with text centered over them
- See Original Image: 
![Capture](https://user-images.githubusercontent.com/63972670/205440366-a4e47002-4334-4e97-af32-2a5ee1baf600.PNG)
-
- See Yours: 
- 
![Capture](https://user-images.githubusercontent.com/63972670/205440420-096537b9-a64a-4530-950c-cd448cacc6cc.PNG)
- Kindly Fix the mistakes.